jfr-microbench/src/main/java/io/netty/handler/jfr/microbench/NoopHandler.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.jfr.microbench;
+
+import io.netty.channel.ChannelDuplexHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelPromise;
+
+import java.net.SocketAddress;
+
+public class NoopHandler extends ChannelDuplexHandler {
+
+    @Override
+    public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
```

Comment:
What is the even the value of this class as it basically overrides everything and does the same as the base class anyway ?

Reply:
I added it for benchmark comparisons. Thought I had documented that, but apparently not. I guess there might already be one that can be used?

Reply:
but why you not just create a `new ChannelDuplexHandler()` and not override anything at all ? It seems odd to me that you override methods and basically just do exactly what the overridden methods do.